charts/latest/azuredisk-csi-driver/templates/csi-azuredisk-node-windows.yaml

```diff
@@ -94,6 +94,7 @@ spec:
             - "--nodeid=$(KUBE_NODE_NAME)"
             - "--metrics-address=0.0.0.0:{{ .Values.node.metricsPort }}"
             - "--drivername={{ .Values.driver.name }}"
+            - "--volume-attach-limit={{ .Values.driver.volumeAttachLimit }}"
```

Comment:
Shouldn't we apply these charts changes to charts/v2.0.0-alpha.1 charts too?

Reply:
Yes. I missed it out for the v2 driver! I have added it.